### ISSUE TITLE
A11y: class “bx--interior-left-nav-collapse__arrow” in component InteriorLeftNav should have associated labels

### ISSUE THREAD

**Issue by xiaolixu** [carbon-design-system/carbon-addons-cloud-react#27]
## Detailed description

> Describe in detail the issue you're having.
class “bx--interior-left-nav-collapse__arrow” in component InteriorLeftNav should have associated labels
> Is this a feature request (new component, new icon), a bug, or a general issue?
bug
> Is this issue related to a specific component?
InteriorLeftNav
> What browser are you working in?
Chrome 68.0.3440.106
> What version of the Carbon Design System are you using?
`    "carbon-addons-cloud": "^2.1.0",
    "carbon-addons-cloud-react": "^1.2.0",
    "carbon-addons-data-viz-react": "^1.21.0",
    "carbon-components": "^9.5.0",
    "carbon-components-react": "6.21.4",
    "carbon-icons": "^7.0.7",
`
> What offering/product do you work on? Any pressing ship or release dates we should be aware of?
Virtual Private Cloud
## Steps to reproduce the issue
use component InteriorLeftNav
